### DISCUSSION TITLE
So many ideas for this new software ...

### DISCUSSION THREAD

**Discussion by virusdunil** [xspanger3770/GlobalQuake#94] [Ideas]
Greetings from quebec

Great software so far, i find it faster than seiscomp to locate earthquakes hehe

Ideas...
Will it run on Linux ?  ( i tried but hehe...)
Add a My Position marker with custom colors on map and set as default view.
different leaflet maps like satellite or terrain ect
show the numbers below stations but at a bit higher zooming
a french translation hehe (with that i could help)
city names when high zoom
more specific on eq location (like if in U.S. ,it just says  United States
Custom alarm sounds
distance slide bar for alarm triggering,  like 500 kms from me
i think its been mentioned but a '"show last 6 - 12 - 24 hours quakes checkbox" would be great

Thats it for now...
Im sure some of them are in your plans hehe just wanted to share my ideas.

**Comment by xspanger3770** [comment]
Hi!
Thanks for all the suggestions, here are a few notes:

- It runs on Linux completely fine, just install the latest Java version and run it using your terminal 
- Current framework supports only polygon rendering and terrain map wouldn't be possible
- Translations are definitely a great idea for the future, although not everything can be translated (for example the displayed locations are fetched from a server that as far as i know only returns English)
- Custom alarm sounds would definitely be great, but I have no experience creating ones - if someone creates ones I'm definitely adding

Other features are either planned or will likely be planned later, and will gradually get added depending on my free time and help from others.

Have a nice day!

**Reply by virusdunil** [reply]
Thanks for your reply...
About linux, i have: java -version
openjdk version "11.0.20" 2023-07-18
OpenJDK Runtime Environment (build 11.0.20+8-post-Ubuntu-1ubuntu122.04)
OpenJDK 64-Bit Server VM (build 11.0.20+8-post-Ubuntu-1ubuntu122.04, mixed mode, sharing)

so,when i start launch command, i get this:
java -jar GlobalQuake_0.9.4.jar
Erreur : LinkageError lors du chargement de la classe principale globalquake.main.Main
        java.lang.UnsupportedClassVersionError: globalquake/main/Main has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0

I do understand i dont have latest java version cuz its on openjdk right ?
i tried purge openjdk then, installed the.deb package: x64 Debian Package on java website.

when i re do the java -version command,it says im still on openjdk....

starting there,as im not a good expert in uptade/remove things in linux well, im stuck and dunno what to do next hehe

**Reply by xspanger3770** [reply]
Yes, the error says that your Java version is outdated.
Try following some online tutorial like this: https://itslinuxfoss.com/how-upgrade-java-ubuntu/#google_vignette
